packages/flutter/test/material/divider_test.dart

```diff
@@ -203,4 +203,10 @@ void main() {
     expect(lineRect.top, dividerRect.top + customIndent);
     expect(lineRect.bottom, dividerRect.bottom - customIndent);
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/39533
+  testWidgets('createBorderSide does not throw exception with null context', (WidgetTester tester) async {
+    Divider.createBorderSide(null);
```

Comment:
consider using `isNot(throwsAssertionError)`

```suggestion
expect(() => Divider.createBorderSide(null), isNot(throwsAssertionError));
```

Reply:
done